packages/orbit-components/src/InputGroup/InputGroup.stories.tsx

```diff
@@ -280,149 +283,106 @@ export const OnChangeBehavior = ({ inputValue }) => {
         />
       </InputGroup>
     </Stack>
-  );
-};
-
-OnChangeBehavior.story = {
-  name: "onChange behaviour",
+  ),
 
   parameters: {
-    info: "Some description about this type of InputGroup in general.",
+    controls: {
+      exclude: [
+        "onChange",
+        "onFocus",
+        "onBlur",
+        "onBlurGroup",
+        "error",
+        "help",
+        "helpClosable",
+        "size",
+        "spaceAfter",
+      ],
+    },
   },
-};
 
-OnChangeBehavior.args = {
-  inputValue: "",
+  args: {
+    inputValue: "",
+  },
 };
 
-export const Disabled = () => {
-  return (
-    <InputGroup disabled label="Disabled">
+export const Disabled: Story = {
+  render: args => (
+    <InputGroup {...args}>
       <InputField placeholder="a" maxLength={11} />
       <InputField placeholder="b" maxLength={11} />
     </InputGroup>
-  );
-};
+  ),
 
-Disabled.story = {
-  name: "Disabled",
+  args: {
+    label: "Disabled",
+    disabled: true,
+  },
 };
 
 const selectOptionsPlayground = [
   { value: 1, label: "First item" },
   { value: 2, label: "Second item" },
 ];
 
-export const Playground = ({
-  label,
-  flex,
-  size,
-  error,
-  help,
-  selectValue,
-  placeholder,
-  inputValue,
-  dataTest,
-  spaceAfter,
-}) => {
-  return (
-    <InputGroup
-      label={label}
-      flex={flex}
-      size={size}
-      onChange={action("onChange")}
-      onFocus={action("onFocus")}
-      onBlur={action("onBlur")}
-      dataTest={dataTest}
-      spaceAfter={spaceAfter}
-    >
+export const Playground: Story = {
+  render: ({ error, help, selectValue, placeholder, inputValue, ...args }) => (
+    <InputGroup {...args}>
       <Select options={selectOptionsPlayground} value={selectValue} error={error} help={help} />
       <InputField placeholder={placeholder} value={inputValue} />
       <Select options={selectOptionsPlayground} value={selectValue} />
       <InputField placeholder={placeholder} value={inputValue} />
     </InputGroup>
-  );
-};
-
-Playground.story = {
-  parameters: {
-    info: "Some description about this type of InputGroup in general.",
+  ),
+
+  args: {
+    label: "Phone number",
+    flex: ["1 0 200px", "1 1 100%", "1 0 150px", "0 1 50%"],
+    size: SIZE_OPTIONS.NORMAL,
```

Comment:
In most of the stories, we have this arg `size`, but I can't see any change when changing from `small` to `normal` or vice versa. Does it make sense to exclude it from controls (in case it really does nothing, maybe I'm just blind :D :D )